jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/view/RssViewerTest.java

```diff
@@ -69,18 +72,32 @@ protected void setUp() {
         topics.add(topic);
         topics.add(topic);
         model.put("topics", topics);
+        List<Topic> topicsWithSpecChars = new ArrayList<>();
```

Comment:
All this code needs to be moved into the test, it doesn't belong to the precondition. You can encapsulate some of it into private methods or use ObjectsFactory to keep the test clean.